hikari/emojis.py

```diff
@@ -235,10 +241,6 @@ def parse(cls, string: str, /) -> UnicodeEmoji:
         UnicodeEmoji
             The parsed UnicodeEmoji object.
         """
-        # Ensure validity.
```

Comment:
Do we even need to keep the parse method if we're not keeping this behaviour around?